### PR TITLE
docs: Add note on extproc header manipulation limitations

### PR DESCRIPTION
**Pull request by davidjumani**
Adds a note about the limitations of header manipulation via ExtProc.

### CODE REVIEW

File: docs/content/guides/traffic_management/extproc/header-manipulation.md [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
 Set up an external processing (ExtProc) server that manipulates request headers for a sample app.
 
 {{% notice note %}}
-External processing is an Enterprise-only feature. 
+External processing is an Enterprise-only feature.
 {{% /notice %}}
 
 {{% notice warning %}}
 Envoy's external processing filter is considered a work in progress and has an unknown security posture. Use caution when using this feature in production environments. For more information, see the [Envoy documentation](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/ext_proc_filter#external-processing).
+As of envoy 1.32, header manipulation via ExtProc only supports adding new headers, overwriting or appending to existing headers.
```
Comment (Rachael-Graham) [inline docs/content/guides/traffic_management/extproc/header-manipulation.md:15]:
```suggestion
As of Envoy 1.32, header manipulation via ExtProc is supported only for adding new headers and for overwriting or appending existing headers.
```

**Review by Rachael-Graham — COMMENTED** [review]
1 comment!

File: docs/content/guides/traffic_management/extproc/header-manipulation.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (davidjumani) [reply docs/content/guides/traffic_management/extproc/header-manipulation.md:15]: I was thinking the same thing. Perhaps @solo-io/solo-docs can advise on this ?

File: docs/content/guides/traffic_management/extproc/header-manipulation.md [hunk 2]
```diff
@@ -1,23 +1,24 @@
...
 Set up an external processing (ExtProc) server that manipulates request headers for a sample app.
 
 {{% notice note %}}
-External processing is an Enterprise-only feature. 
+External processing is an Enterprise-only feature.
 {{% /notice %}}
 
 {{% notice warning %}}
 Envoy's external processing filter is considered a work in progress and has an unknown security posture. Use caution when using this feature in production environments. For more information, see the [Envoy documentation](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/ext_proc_filter#external-processing).
+As of Envoy 1.32, header manipulation via ExtProc is supported only for adding new headers and for overwriting or appending existing headers.
```
Comment (Rachael-Graham) [inline docs/content/guides/traffic_management/extproc/header-manipulation.md:15]:
```suggestion
As of Envoy 1.32, header manipulation via ExtProc is supported only for adding new headers and for overwriting or appending existing headers. Note that the [`append_action`](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/base.proto#envoy-v3-api-msg-config-core-v3-headervalueoption) is not supported.
```

**Review by Rachael-Graham — APPROVED** [review]

**Review by andy-fong — APPROVED** [review]

**Review by Rachael-Graham — APPROVED** [review]

**Pull request merged**